web/sites/default/config/system.cron.yml

```diff
@@ -1,6 +1,6 @@
 threshold:
   requirements_warning: 172800
   requirements_error: 1209600
-logging: 0
+logging: 1
```

Comment:
Is this intentional and okay to enable?

Reply:
I want to turn on cron logging long enough to get that up in production and see our cron times there for a couple of days. It adds about 10 lines to each cron run, so crazy, but not too crazy. :)